include/ensmallen_bits/ens_version.hpp

```diff
@@ -34,10 +38,14 @@ struct version
   static inline std::string as_string()
   {
     const char* nickname = ENS_VERSION_NAME;
+    const char* year = ENS_VERSION_YEAR;
+    const char* month = ENS_VERSION_MONTH;
+    const char* day = ENS_VERSION_DAY;
 
     std::stringstream ss;
     ss << version::major << '.' << version::minor << '.' << version::patch
-       << " (" << nickname << ')';
+       << " (" << nickname << ")\n" 
+       << "Released on " << year << '-' << month << '-' << day;
```

Comment:
I feel like someone would expect `ens::version::as_string()` to return something very simple without a newline; if we do go with this idea, maybe we should make an additional function `date()` that returns `year << '-' << month << '-' << day`?